infinity/custom_items/general/shraken-Harnes-custom_saare.json

```
@@ -1,6 +1,8 @@
 {
-  "ckey" :            "jacking",
-  "character_name" :  "Mammoth Nestor",
+  "ckey" :            "shraken",
+  "character_name" :  "Ethan Harsen",
   "item_path" :       "/obj/item/clothingbag/custom_saare",
   "req_access" :      [ "ACCESS_SECURITY" ]
-  }
\ No newline at end of file
+   }
+
+
```

```suggestion
   }

```
Убери восьмую строку и убедись, что седьмая абсолютно пустой.
После фигурной скобки должна быть одна пустая строка